pleiades/vocabularies/browser/controlpanel.py

```diff
@@ -0,0 +1,32 @@
+from plone.app.registry.browser import controlpanel
+from collective.z3cform.datagridfield import DataGridFieldFactory
+
+from ..interfaces import IPleiadesSettings
+
+
+class PleiadesSettingsEditForm(controlpanel.RegistryEditForm):
+
+    schema = IPleiadesSettings
+    label = u"Pleiades Site Settings"
+    description = u""
+    enableCSRFProtection = True
+
+    def updateFields(self):
+        super(PleiadesSettingsEditForm, self).updateFields()
+        self.fields['time_periods'].widgetFactory = DataGridFieldFactory
+        self.fields['time_periods'].allow_insert = True
+
+    def updateWidgets(self):
+        super(PleiadesSettingsEditForm, self).updateWidgets()
```

Comment:
They will. I meant that the `updateWidgets` function that only calls `super` is unnecessary.